packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -22,23 +86,76 @@ const TeamPromptMeeting = (props: Props) => {
     graphql`
       fragment TeamPromptMeeting_meeting on TeamPromptMeeting {
         ...TeamPromptTopBar_meeting
+        ...useMeeting_meeting
+        phases {
+          stages {
+            ... on TeamPromptResponseStage {
```

Comment:
-1 Could you please rename `TeamPromptResponsesPhase` to `TeamPromptResponsePhase`? (notice  the missing "s"). not a problem in this PR, but i didn't discover it until now

Reply:
no strong opinions here, the origin of the name is the fact that this phase is for all responses, not one. let me know if that makes sense. if you think it's still better to use `TeamPromptResponsePhase` I will fix

Reply:
ah gotcha, that works!